### ISSUE TITLE
Calculate copy number based on log2 ratio in plots

### ISSUE THREAD

**Issue by maehler** [hydra-genetics/reports#90]
The copy number can be calculated based on the log<sub>2</sub> ratios in the plots. This will need to make a few assumptions regarding the expected ploidy of each chromosome. Since the report as of writing doesn't have information on the sex assignment of the sample, the sex chromosomes will have to be handled carefully in order to not mislead users.

If $v$ is the log<sub>2</sub> ratio, $n$ is the number of copies, and $p$ is the reference ploidy, then $v = \log_2 \frac{n}{p}$. By rearranging, the copy number is then $n = p \cdot 2^v$. This assumes that the tumor purity is 100%, and this would be the first step. Later we can adjust the log<sub>2</ratio> with an estimate of the tumor purity, but I will treat that separately.

**Comment by maehler** [comment]
The copy number could be presented with a secondary axis, but the scale of this might become tricky to get right.

**Comment by jonca79** [comment]
I'm not sure what is what is easy to implement but here are some options:
1. Use the lines / scale of log2 ratio and only show copy number for these numbers.
2. Show only integer numbers specified by some fixed integer tick units (example 1,2,3,4,6,8,10,20,50,100)
I would prefer the second option. Seems possible to do at least for some graphics packages.

**Comment by maehler** [comment]
Yeah, 1. was my first thought, but 2. would be easier to read. Don't think that this should be too difficult to implement.

**Comment by maehler** [comment]
The secondary axis for copy numbers has been implemented in #93. One thing that I'm not quite sure how to handle is the ploidy for the sex chromosomes. It's probably generally safe to assume that the ploidy of the autosomes is 2, but for the sex chromosomes it depends on the sex of the sample being analysed, and this is not information that is currently being passed to the report generation. Is there anything obvious that I'm overlooking here? Could it be an option to just assume a ploidy of 2 across the whole genome and state this clearly as a warning, at least initially?
